packages/RNTester/js/utils/RNTesterNavigationContext.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+'use strict';
+
+const RNTesterList = require('./RNTesterList');
+
+export type RNTesterNavigationState = {openExample: ?string, ...};
+
+function RNTesterNavigationReducer(
```

Comment:
Any specific reason why this was moved from RNTesterNavigationReducer to RNTesterNavigationContext? @stealthanthrax

Reply:
@manyaagarwal , what do you mean by `this`?

Reply:
All the code in this file

Reply:
@manyaagarwal , so that all the AsyncStorage interaction takes place from one file.

Reply:
@stealthanthrax This is the same code from `RNTesterNavigationReducer`, there has been no addition. Is there something else also that you intend to add to this file?